lib/core/presentation/pages/home/views/list/home_newsfeed_list.dart

```diff
@@ -18,8 +17,6 @@ class HomeNewsfeedListView extends StatelessWidget {
     final t = Translations.of(context);
     final colorScheme = Theme.of(context).colorScheme;
     final bloc = context.read<NewsfeedListingBloc>();
-    final refreshController = RefreshController();
-
     return BlocConsumer<NewsfeedListingBloc, NewsfeedListingState>(
```

Comment:
I tried put refreshController into BloC. It causing error that print out refreshController is using multiple times. Please put it back into UI